distribution/lib/Standard/Image/0.0.0-dev/src/Data/Matrix.enso

```diff
@@ -1,4 +1,4 @@
-from Standard.Base import all
+from Standard.Base import all hiding Sort_Direction
```

Comment:
should be solved by moving Sort_Direction from Ordering.

Reply:
Done in 78375fa2cacf99e76cd574ce3943911b5c2a0ad8